components/progressTracker/ProgressTracker.js

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import Box from '../box';
+import ProgressStep from './ProgressStep';
+
+class ProgressTracker extends PureComponent {
+  render() {
+    const { color, children, currentStep, allStepsCompleted } = this.props;
+
+    const classNames = cx(theme['tracker']);
+
+    return (
+      <Box className={classNames}>
+        {React.Children.map(children, (child, index) => {
+          const activeStep = Math.max(0, currentStep);
+
+          return (
+            <ProgressStep
+              { ...child.props} 
```

Comment:
I think you forgot to lint, or is this space allowed?

Reply:
that's weird, the precommit hook should catch this

Reply:
What an eagle eye, but yes. It should have been caught by the pre-commit hook, will check that out now.

 :ok: by @Kemosabert on 19/06/2018 17:40:34